### ISSUE TITLE
Do we have a permanent `version` function for AstroPack which I can use?

### ISSUE THREAD

**Issue by EastEriq** [EranOfek/AstroPack#455]
Context: https://github.com/PolishookDavid/LAST_OCS/issues/28

Problem: record together with the images observed the version identifiers of _all_ software components which contributed to the process. Among them, we should store the version/tag of Astropack in use for acquisition.

For the various classes I maintain, which inherit from LAST_Handle, I have created an utility function https://github.com/EastEriq/LAST_Handle/blob/mastrolindo/%2Bobs/%2Butil/%2Btools/getgitversion.m, and solved the problem for all my instument and abstract classes. I'm just missing AstroPack.

It will be trivial for me to call such a function also against AstroPack on creation of unitCS, so to store also that key. For that I would need an absolute path to a file of the AstroPack repo (robust to all vagaries of installation, perhaps derived from the startup script), or the name of an AstroPack function which is immutable throughout the life of the universe (I could use `which`, i.e. `which('celestial.time.julday')`). What would you suggest?

**Comment by EranOfek** [comment]
AstroPack version is stored in proc images by pipeline

**Comment by EastEriq** [comment]
And that is not good, because it can be different from the version of AstroPack which acquired the images. No? Haven't we just worked with acquisition on `dev1` with pipeline which needed to be on `v2.1`? Haven't we just committed today a fix to astropacK which affected the application of the pointing model to the acquired images? Will it never happen that you reprocess a raw image with a newer version of AsTroPaCK?
